### PR TITLE
ELMU-9 Update educandu and introduce an articles controller

### PR DESCRIPTION
**Pull request by laurentiu-ilici**
Closes: https://educandu.atlassian.net/browse/ELMU-9

### CODE REVIEW

**Review by ahelmberger — APPROVED** [review]

File: src/articles-controller.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  static get inject() { return [DocumentStore]; }
+
+  constructor(documentStore) {
+    this.documentStore = documentStore;
+  }
+
+  registerPages(router) {
+    router.get('/articles/*', async (req, res) => {
+      const slug = req.params[0] || '';
+      const doc = await this.documentStore.findOne({ namespace: 'articles', slug });
```
Comment (adrianaluchian) [inline src/articles-controller.js:17]: Small thing, but in order for us not to need to change this again when we remove namespaces, can we just find by slug here?

Comment (laurentiu-ilici) [reply src/articles-controller.js:17]: Hmmm, not sure, this was the old signature how it was finding the documents... @ahelmberger

Comment (ahelmberger) [reply src/articles-controller.js:17]: Good idea :+1:

Comment (laurentiu-ilici) [reply src/articles-controller.js]: Done

**Pull request merged**